float64[] stateMeasured
float64[] covarMeasured 
